client/src/test/java/com/aws/greengrass/cli/commands/DeploymentCommandTest.java

```diff
@@ -174,6 +176,25 @@ void GIVEN_WHEN_components_runwith_provided_THEN_request_contains_the_info() thr
         assertThat(exitCode, is(0));
     }
 
+    @Test
+    void GIVEN_WHEN_failure_handling_policy_is_provided_THEN_request_contains_provided_failure_handling_policy() {
+        int exitCode = runCommandLine("deployment", "create", "-p", FAILURE_HANDLING_POLICY.getValue());
+
+        CreateLocalDeploymentRequest request = new CreateLocalDeploymentRequest();
+        request.setFailureHandlingPolicy(FAILURE_HANDLING_POLICY);
+        verify(nucleusAdapteripc).createLocalDeployment(request);
+        assertThat(exitCode, is(0));
+    }
+
+    @Test
+    void GIVEN_WHEN_failure_handling_policy_provided_with_invalid_words_THEN_invalid_request_is_returned() {
+        int exitCode =
+                runCommandLine("deployment", "create", "-p", "Hello World");
```

Comment:
What error message do customers see today?

Reply:
Invalid value for option '--failure-handling-policy': expected one of [ROLLBACK, DO_NOTHING] (case-sensitive) but was 'test'